functions/.gitignore

```diff
@@ -1 +1,2 @@
-node_modules/
\ No newline at end of file
+node_modules/
+.secret.local
```

Comment:
Since it's not obvious from the file name, i would recommend adding a comment to explain what this is for

Reply:
https://github.com/ably/asset-tracking-backend-demo/pull/11/commits/47937550edb5ec0e6d51e829ba5cc6095cf30a5f